packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -377,9 +377,26 @@ const defaultConfigs = {
     ]
   },
   DEFI_PULSE_TOTAL_TVL: {
-    type: "defipulsetotal",
+    type: "defipulse",
     lookback: 604800,
-    minTimeBetweenUpdates: 600
+    minTimeBetweenUpdates: 600,
+    project: "all"
+  },
+  DEFI_PULSE_SUSHI_TVL: {
+    type: "defipulse",
+    lookback: 604800,
+    minTimeBetweenUpdates: 600,
+    project: "all"
```

Comment:
This should be `sushiswap` or something like that right? Same with Uniswap below

Reply:
Since the type="defipulse" is used in CreatePriceFeed.js to identify the use of DefiPulsePriceFeed, shouldn't the type stay as "defipulse" for both suhi and uni?

Reply:
i think he means project:"all" should be project:"SushiSwap"? ditto with uni

Reply:
Ah got ya. Made change.